### PR TITLE
RHMAP-11433 - RHMAP-11432 Add ESLint into the Project without any impact 

### PR DESCRIPTION
**Pull request by camilamacedo86**
RHMAP-11433 - Add lint step to current job - https://issues.jboss.org/browse/RHMAP-11433

### CODE REVIEW

**Comment by fh-automatron** [comment]
Can one of the admins verify this patch?

**Comment by fh-automatron** [comment]
Can one of the admins verify this patch?

**Comment by grdryn** [comment]
@camilamacedo86 It looks like your branch is out of date, even though it has been a couple of days since anything changed on master. Could you rebase it please? Once it's rebased, I'll get the build job to trigger. :)

**Comment by camilamacedo86** [comment]
@grdryn Done :-)

**Comment by grdryn** [comment]
Test this please.

**Comment by grdryn** [comment]
^That comment was just to trigger the jobs

File: Gruntfile.js [hunk 1]
```diff
@@ -79,7 +88,7 @@ module.exports = function(grunt) {
   grunt.loadNpmTasks('grunt-jsxgettext');
   grunt.loadNpmTasks('grunt-zanata-js');
 
-  grunt.registerTask('test', ['fh:test']);
+  grunt.registerTask('test', ['fh:test', 'eslint']);
```
Comment (grdryn) [inline Gruntfile.js]: I'm not sure why, but this doesn't seem to be running as part of the job ('test' is, but 'eslint' isn't). Does eslint run if you run `grunt test` locally?

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    "Datamap": true,
+    "d3": true,
+    "angular": true,
+    "$": true,
+    "jQuery": true,
+    "xssFilters": true
+  },
+  "extends": "eslint:recommended",
+  "rules": {
+    // TODO remove line "no-unused-vars"
```
Comment (grdryn) [inline .eslintrc.json]: I'd suggest removing the comments, since that makes it invalid JSON.

**Comment by grdryn** [comment]
The build is failing because it can't do something wrt zanata it seems.

**Comment by grdryn** [comment]
Retest this please.

File: Gruntfile.js (hunk 1 above)
Comment (camilamacedo86) [inline Gruntfile.js]: It is the same way that works into NGUI. I think we should do it as standard.

File: .eslintrc.json (hunk 2 above)
Comment (camilamacedo86) [inline .eslintrc.json]: It is working as well and these are the points that we need work in the other tasks to finish it. Do you think is better remove the TODO?

**Comment by camilamacedo86** [comment]
@grdryn Could you check now?

**Comment by grdryn** [comment]
add to whitelist

**Comment by grdryn** [comment]
@camilamacedo86 any time you update the PR now with a new commit or something, it'll trigger the job.

**Comment by jasonmadigan** [comment]
test this please

**Comment by jasonmadigan** [comment]
test this please

**Comment by camilamacedo86** [comment]
Test this please.

**Comment by camilamacedo86** [comment]
Retest this please.

**Pull request merged**